intelligent-video-recording/gst_display.py

```diff
@@ -106,29 +105,51 @@ def parseLabels(self, file):
     def newPrediction(self, element, meta):
         data = json.loads(meta)
 
-        # Parse class id from prediction
```

Comment:
Document this newPrediction function. Put a comment explaining how is it being used, what is generated by the inferencebin element and when. Remember this demo is intended to show how GstInference works for someone who has never heard about it